aws/network-primitives/scripts/create_domain_node_compose_file.sh

```diff
@@ -103,23 +103,20 @@ services:
     command: [
       "--chain", "\${NETWORK_NAME}",
       "--base-path", "/var/subspace",
-      "--execution", "wasm",
-#      "--enable-subspace-block-relay",
       "--state-pruning", "archive",
       "--blocks-pruning", "archive",
-      "--listen-addr", "/ip4/0.0.0.0/tcp/30333",
+      "--listen-on", "/ip4/0.0.0.0/tcp/30333",
       "--dsn-external-address", "/ip4/$EXTERNAL_IP/udp/30433/quic-v1",
       "--dsn-external-address", "/ip4/$EXTERNAL_IP/tcp/30433",
-#      "--piece-cache-size", "\${PIECE_CACHE_SIZE}",
       "--node-key", "\${NODE_KEY}",
-      "--rpc-cors", "all",
-      "--rpc-external",
       "--in-peers", "500",
       "--out-peers", "250",
       "--in-peers-light", "500",
       "--rpc-max-connections", "10000",
-      "--prometheus-port", "9615",
-      "--prometheus-external",
+      "--rpc-cors", "all",
+      "--rpc-listen-on",
+      "--rpc-methods", "auto",
```

Comment:
`auto` is the default, either remove it or use this explicitly:
```suggestion
      "--rpc-methods", "safe",
```